src/components/BoardPiece.ts

```diff
@@ -77,7 +82,7 @@ export class BoardPiece {
   /**
    * Map of piece to background image CSS class name.
    */
-  private static PIECE_CLASS_MAP: Record<Side, Record<PieceType, string>> = {
+  public static PIECE_CLASS_MAP: Record<Side, Record<PieceType, string>> = {
```

Comment:
Since this was private before, I guess you don't really need it and can remove it entirely right?